packages/vhd-format/vhd-format.0.13.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+maintainer: "dave@recoil.org"
+authors: ["Dave Scott" "Jon Ludlam"]
+tags: ["org:mirage" "org:xapi-project"]
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+homepage: "https://github.com/mirage/ocaml-vhd"
+doc: "https://mirage.github.io/ocaml-vhd/"
+bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
+depends: [
+  "ocaml" {>= "4.10.0"}
+  "bigarray-compat" {>= "1.1.0"}
+  "cstruct" {>= "6.0.0"}
+  "io-page"
+  "result" {>= "1.5"}
+  "rresult" {>= "0.3.0"}
```

Comment:
```suggestion
  "rresult" {>= "0.6.0"}
```
I think this may be the culprit

Reply:
I thought I pushed the change, but apparently I missed it. I believe the lower bound should be 0.7.0